roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -1,3 +1,11 @@
 ---
 # defaults file for openshift_prometheus
+openshift_prometheus_install_prometheus: True
+
 kubeconfig: "{{ openshift.common.config_base }}/master/openshift-master.kubeconfig"
+openshift_prometheus_namespace: prometheus
```

Comment:
we should confirm with @smarterclayton or @twiest what this should be named.  There was talk at one time of having infra namespaces like 'openshift-infra-prometheus'